noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/components/tail_to_public_output_composer/split_to_public.nr

```diff
@@ -10,8 +10,6 @@ pub fn split_to_public(
     data: PrivateAccumulatedDataBuilder,
     min_revertible_side_effect_counter: u32
 ) -> (PublicAccumulatedData, PublicAccumulatedData) {
-    assert(min_revertible_side_effect_counter != 0, "min_revertible_side_effect_counter must not be 0");
```

Comment:
Removed this check as this is an unconstrained function and the same check is done in `previous_kernel_validator.nr`.